<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>
  
  <node name="stepper_left" pkg="phidget_stepper" type="stepper.py" output="screen">
    <rosparam param="isLeft">true</rosparam>
    <rosparam param="serialId">420520</rosparam>
  </node>

  <node name="stepper_right" pkg="phidget_stepper" type="stepper.py" output="screen">
    <rosparam param="isLeft">false</rosparam>
    <rosparam param="serialId">420519</rosparam>
  </node>

</launch>
